### PR TITLE
Informaticadatamanagement dot net SDK beta

### PR DESCRIPTION
**Pull request by almasak**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -989,6 +989,13 @@
 # ServiceLabel: %Spring App Discovery %Mgmt
 # ServiceOwners:                                                   @sunkun99
 
+# PRLabel: %Informaticadatamanagement
```
Comment (jsquire) [inline .github/CODEOWNERS:992]:
```suggestion
# PRLabel: %Informatic Data Management
```

You cannot create arbitrary new labels.   Please use "Informatic Data Management", which I've created on your behalf.

Comment (jsquire) [inline .github/CODEOWNERS:992]: Please move this section up to ~L950 to preserve alphabetical ordering by label name in the management section.

**Review by jsquire — DISMISSED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (almasak) [reply .github/CODEOWNERS:992]: @jsquire I am assuming you have created the new label as _"Informatica Data Management"_, and not as _"Informatic Data Management"_

Comment (jsquire) [reply .github/CODEOWNERS:992]: Yes.  that's a typo.   Apologies.

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -948,8 +948,7 @@
 # ServiceOwners:                                                   @davidemontanari @atastrophic @marcodalessandro @rohankhandelwal @riteshrao
 
 # PRLabel: %Informatica Data Management
-/sdk/informaticadatamanagement/Azure.ResourceManager
-.Informaticadatamanagement/                                        @almasak
+/sdk/informaticadatamanagement/Azure.ResourceManager.Informaticadatamanagement/ @almasak
```
Comment (jsquire) [inline .github/CODEOWNERS:951]:
```suggestion
/sdk/informaticadatamanagement/Azure.ResourceManager*/ @almasak
```

We can shorten the path and still achieve the desired effect.

Comment (almasak) [reply .github/CODEOWNERS]: Good suggestion, I have made the change.

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -950,7 +950,7 @@
 # PRLabel: %Informatica Data Management
 /sdk/informatica/Azure.ResourceManager.*/            @almasak
 
-# ServiceLabel: %informatica %Mgmt
+# ServiceLabel: %Informatica %Mgmt
```
Comment (jsquire) [inline .github/CODEOWNERS:953]:
```suggestion
# ServiceLabel: %Informatica Data Management %Mgmt
```

**Review by jsquire — APPROVED** [review]

File: .github/CODEOWNERS (hunk 3 above)
Comment (jsquire) [reply .github/CODEOWNERS:953]: Without the full label, we won't see support issues route correctly.

**Review by ArthurMa1978 — APPROVED** [review]

**Pull request merged**